extensions/integration-tests/src/main.ts

```diff
@@ -12,16 +12,21 @@ const TEST_SETUP_COMPLETED_TEXT: string = 'Test Setup Completed';
 const EXTENSION_LOADED_TEXT: string = 'Test Extension Loaded';
 const ALL_EXTENSION_LOADED_TEXT: string = 'All Extensions Loaded';
 
+const logPrefix: string = 'test:int:main';
+const debug: any = require('debug')(logPrefix);
+const trace: any = require('debug')(`${logPrefix}:trace`);
 let statusBarItemTimer: NodeJS.Timer;
 
 export function activate(context: vscode.ExtensionContext) {
 	let statusBarItem = vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Left);
 	vscode.commands.registerCommand('test.setupIntegrationTest', async () => {
-		let extensionInstallersFolder = normalize(join(__dirname, '../extensionInstallers'));
+		let extensionInstallersFolder = normalize(join(__dirname, '../../extensionInstallers'));
```

Comment:
Why is this changing? I don't see any changes that would cause the outdir path to be changed?

Reply:
with the outDir specified the code gets dropped in out\src and when no outDir was specified, it gets dropped directly under 'out'. So the generated code is now one directory deeper.